### PR TITLE
chore: move EntityStore into core and remove visual-sdk package

### PR DESCRIPTION
**Pull request by elylucas**

### CODE REVIEW

File: packages/core/src/entity/constants.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export enum PostMessageMethods {
+  REQUEST_ENTITIES = 'REQUEST_ENTITIES',
+  REQUESTED_ENTITIES = 'REQUESTED_ENTITIES',
+}
```
Comment (primeinteger) [inline packages/core/src/entity/constants.ts:4]: Looks like this same enum is defined in `packages/core/src/constants.ts` as well

**Review by primeinteger — APPROVED** [review]
Looking good!  Found one small thing mentioned below

File: packages/core/src/entity/constants.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export enum PostMessageMethods {
```
Comment (Chaoste) [inline packages/core/src/entity/constants.ts:1]: Not necessary to be done in this PR but just wanna call out that we should avoid enums as long as they are not necessary. Our [TS standards doc](https://contentful.atlassian.net/wiki/spaces/ENG/pages/4189094335/TypeScript+Standards#Enums) recommends using union types or object literals.

**Review by Chaoste — APPROVED** [review]
Great to see that we're dropping again the `visual-sdk` 👍🏻

**Pull request merged**